lib/elixir/lib/file.ex

```diff
@@ -607,8 +607,6 @@ defmodule File do
         [dest|acc]
       {:error, :eexist} ->
         if callback.(src, dest) do
-          # If rm/1 fails, copy/2 will fail
```

Comment:
Your comment is true here, because the previous copy was ran with exclusive mode.